package.json

```diff
@@ -18,7 +18,8 @@
 		"create-rule": "node ./scripts/create-rule.js",
 		"run-rules-on-codebase": "node ./test/run-rules-on-codebase/lint.mjs",
 		"integration": "node ./test/integration/test.js",
-		"smoke": "eslint-remote-tester --config ./test/smoke/eslint-remote-tester.config.js"
+		"smoke": "eslint-remote-tester --config ./test/smoke/eslint-remote-tester.config.js",
+		"update-rules": "node ./scripts/update-rules.js"
```

Comment:
`update-rules` is not exactly describing what it does.

Reply:
Renamed to `update-rules-in-readme`.